wordpress-mobile/WordPress-iOS
# [Gutenberg] Send http status code to network error callback.

`gutenberg-mobile` side PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1686

This PR ensures that the real http status code is returned on the network error callback.

To test:
- Run metro server on https://github.com/wordpress-mobile/gutenberg-mobile/pull/1686
- Check test steps on https://github.com/wordpress-mobile/gutenberg-mobile/pull/1686

PR submission checklist:
